rules-tests/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector/FixtureAlwaysSprintf/encapsed_strings_to_sprintf_should_escape_percent.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\Tests\CodingStyle\Rector\Encapsed\EncapsedStringsToSprintfRector\Fixture;
```

Comment:
should use `FixtureAlwaysSprintf` subnamespace

```suggestion
namespace Rector\Tests\CodingStyle\Rector\Encapsed\EncapsedStringsToSprintfRector\FixtureAlwaysSprintf;
```

on other files in this directory as well